src/main/webapp/app/admin/admin.route.ts

```diff
@@ -13,6 +13,7 @@ import { StatisticsComponent } from 'app/admin/statistics/statistics.component';
 import { DocsComponent } from 'app/admin/docs/docs.component';
 import { organizationMgmtRoute } from 'app/admin/organization-management/organization-management.route';
 import { MetricsComponent } from 'app/admin/metrics/metrics.component';
+import { IrisSettingsUpdateRoutingModule } from 'app/iris/settings/iris-settings-update-routing.module';
```

Comment:
apparently unused according to github

Reply:
Was probably a remnant, when I initially did not lazy load it 🤷🏻